### PR TITLE
Add wafflefries

### PR DESCRIPTION
**Pull request by Shane4666**
Please review this, @casey-collab

### CODE REVIEW

File: english/wafflefries/wafflefries.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+You sit down at the local bar and find some fries, the fries are given
```
Comment (casey-collab) [inline english/wafflefries/wafflefries.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Shane4666** [comment]
Please review this, @casey-collab

File: english/wafflefries/wafflefries.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+You sit down at the local bar and find some fries, the fries are given.
```
Comment (casey-collab) [inline english/wafflefries/wafflefries.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**